src/base.css

```diff
@@ -1,8 +1,34 @@
-html, body {
+html, body, a {
   font-size: 14px;
   font-family: sans-serif;
 }
 
+a {
+  color: #5193dd;
+  text-decoration: none;
+  border-radius: 5px;
```

Comment:
I'd like to keep all dimensions relative, rather than specific pixels.  See examples below that use `em` units.